Update Lido contracts calldata info

alx-khramov opened the pull request:
This PR updates ERC-7730 clear signing information for some Lido contracts.
The code was generated using https://get-clear-signed.ledger.com/ and subsequently edited manually.
Contract addresses reference: https://docs.lido.fi/deployed-contracts/

paoun-ledger commented on registry/lido/calldata-WithdrawalQueueERC721.json:
```diff
@@ -0,0 +1,259 @@
+        "fields": [
+          {
+            "label": "Amount to withdraw",
+            "format": "tokenAmount",
```
you probably need to add the token address, or the device won't be able to display correctly that amount with a ticker and magnitude.
Something like that:

          {
            "label": "Amount to withdraw",
            "format": "tokenAmount",
            "params": { "token": "$.metadata.constants.stETHaddress" },
            "path": "#._amounts.[]"
          },

paoun-ledger replied:
as an example, if the device doesn't have the full context, it will display:

    Amount to withdraw
    10000000000000000000 ???

instead of

    Amount to withdraw
    10 stETH

paoun-ledger commented on registry/lido/calldata-WithdrawalQueueERC721.json:
```diff
@@ -0,0 +1,259 @@
+            "path": "#._owner"
+          },
+          {
+            "path": "#._permit",
```
you can remove completely that block if the permit don't have to be shown to the user

paoun-ledger commented on registry/lido/calldata-stETH.json:
```diff
@@ -3,38 +3,71 @@
+          },
+          {
+            "label": "Amount",
+            "format": "tokenAmount",
```
here again you could add the token address, which is the the recipient of the transaction in this case so something like that: `"params": { "tokenPath": "@.to" }`

alx-khramov commented:
@paoun-ledger Hey! Sorry for the delay. I’ve addressed all your comments:
- Added the token addresses
- Made sure all labels are within the 30-character limit
